scripts/register-cluster-with-dynatrace.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env sh
+
+VAULT_NAME=$1
+DYNATRACE_INSTANCE=$2
+
+DYNATRACE_CLUSTERROLE_BINDING=roles/dynatrace-cluster-role-binding.yaml
+
+error_exit()
+{
+	echo "$1" 1>&2
+	exit 1
+}
+
+kubectl apply -f ${DYNATRACE_CLUSTERROLE_BINDING}
+
+if kubectl config current-context; then
+  K8S_API_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+  CLUSTER_NAME=$(kubectl config view --minify -o jsonpath='{.clusters[0].name}')
+	BEARER_TOKEN=$(kubectl get secret $(kubectl get sa dynatrace-monitoring \
```

Comment:
tabs and spaces are mixed here and the next two lines

Reply:
strange it looks fine in the IDE

Reply:
IDE won't show it by default

![image](https://user-images.githubusercontent.com/21194782/86451797-523d5180-bd13-11ea-9d7f-69502aad7d85.png)

still looks broken